index.bs

```diff
@@ -1214,15 +1214,17 @@ Gets the {{DOMException}} object passed to {{SensorErrorEventInit}}.
 <div algorithm="construct sensor object">
 
     : input
+    :: |sensorInterface|, an [=identifier=] of the [=interface=]
```

Comment:
sensor_interface

And the same elsewhere to be consistent with the naming convention within the spec.